### ISSUE TITLE
Boat Passes Glitched out

### ISSUE THREAD

**Issue by xMurderMelody** [Wynncraft/Issues#7447]
Hey There!
I have to tell you about something that i think/know isnt suppost to be like that. I was in Selchar today because of a Quest. I got there by swimming with my boat, but i didnt want to swim back so i got myself a Boat Pass. And then when i wanted to use it and throw it in the hopper of the Boat, it switched places in my Inventory with the Item i had there before. after that the Pass went invisable and the Captain said his lines he says when you throw in an Item that isnt a Boat Pass. After that as i talked to the Boat Pass Seller again, it appeared im my Inventory again. So i tried it again but this time i put it on the place in my Hotbar where i normally have my Horse. And guess what happened. It did its invisable thing again and it TOOK MY HORSE instead! And again the Captain said his lines. Like before: The Passes appeared in my Inventory, but my horse was GONE. I had a black horse lv 20 wich glitched away... great. I maybe have some screenshots of my Horse, but not of the Moment it happened because it was the last thing i thought would happen. Please, if you have a way of giving me my horse back, let me know. I can even send the screenshots of me having the horse as a proof, if i need to. My In-Game name is xMurderMelody. This is also my first post on the Forum so excuse me if i wrote something not the way its suppost to be (im German).
Anyways, have a nice day.
-xMurderMelody

**Comment by thecoolkogut4** [comment]
Hey there, unfortunately Wynncraft enforces a strict no-refund policy. That means you won't be receiving your items back. However, thank you for reporting the bug.

**Comment by Numbermaniac** [comment]
Is it possible you were lagging a bit? Perhaps Wynncraft didn't register you switching the boat pass with the horse in your inventory in time, due to internet lag.

**Comment by xMurderMelody** [comment]
i dont think so, because i tried to switch other Items and there was no problem with that.

**Comment by TheMostEpic01** [comment]
ummmm a glitch caused him to loose the horse. how is that unable to be given because of a no refund policy if he never sold it but it glitched away.

**Comment by thecoolkogut4** [comment]
According to dictionary.com, refund is "to give back or restore (especially money)"

In this case, you want the horse back instead of money. Yes, xMurderMelody never sold the horse to the hopper (xMurderMelody just dropped it in accidentally). If you want it back it would be considered a refund (to give back the horse). The server took the horse away (the hopper), so in order to get it back the server (staff) need to give you the horse. Therefore it would be considered a refund.
